### PR TITLE
restructure packages

### PR DESCRIPTION
**Pull request by kortemik**

### CODE REVIEW

**Comment by kortemik** [comment]
closes #130

**Review by eemhu — APPROVED** [review]
lgtm

File: src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java [hunk 1]
```diff
@@ -122,7 +122,7 @@ public void run() {
                     if (LOGGER.isDebugEnabled()) {
                         LOGGER.debug("resuming buffer <{}>, activeBuffers <{}>", activeBuffers.get(0), activeBuffers);
                     }
-                    complete = innerLoop(relpFrame, true);
+                    complete = innerLoop(relpFrame);
```
Comment (StrongestNumber9) [inline src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java:125]: Intended change in this PR?

File: src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java [hunk 2]
```diff
@@ -135,7 +135,7 @@ public void run() {
                         return; // TODO this is quite ugly return, single point of return is preferred!
                     }
 
-                    if (innerLoop(relpFrame, false)) {
+                    if (innerLoop(relpFrame)) {
```
Comment (StrongestNumber9) [inline src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java:138]: Intended change in this PR?

File: src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java [hunk 3]
```diff
@@ -164,7 +164,7 @@ public void run() {
         }
     }
 
-    private boolean innerLoop(RelpFrameLeaseful relpFrame, boolean hasRef) {
+    private boolean innerLoop(RelpFrameLeaseful relpFrame) {
```
Comment (StrongestNumber9) [inline src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java:167]: Intended change in this PR?

File: src/main/java/com/teragrep/rlp_03/EventLoop.java [hunk 4]
```diff
@@ -95,10 +97,6 @@ private void registerPendingRegistrations() {
         }
     }
 
-    public Selector selector() {
-        return selector;
-    }
-
```
Comment (StrongestNumber9) [inline src/main/java/com/teragrep/rlp_03/EventLoop.java:101]: Intended change in this PR?

File: src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java (hunk 2 above)
Comment (kortemik) [reply src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java:138]: yes

File: src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java (hunk 3 above)
Comment (kortemik) [reply src/main/java/com/teragrep/rlp_03/channel/context/RelpReadImpl.java:167]: yes

File: src/main/java/com/teragrep/rlp_03/EventLoop.java (hunk 4 above)
Comment (kortemik) [reply src/main/java/com/teragrep/rlp_03/EventLoop.java:101]: yes

**Pull request merged**